lore/stores/disk.py

```diff
@@ -32,6 +41,14 @@ def __getitem__(self, key):
 
     def __setitem__(self, key, value):
         with timer('write %s' % key):
+            if isinstance(value, pandas.core.frame.DataFrame):
```

Comment:
Has datetime compatibility in dataframes always been broken? Would be good to get a test case around this.

Reply:
No, this is exclusively related to snowflake. Pickling dataframes returned from snowflake is flaky. It works at times, but fails at other times.